### DISCUSSION TITLE
Custom Resources (CRD) file is not showing on AKS cluster after installing Keda

### DISCUSSION THREAD

**Discussion by sanprasingh** [kedacore/keda#1110] [Q&A / Need Help] [answered]
Hi Team,
Below Custom Resources (CRD) file is not showing on AKS cluster after installing Keda.
scaledobjects.keda.k8s.io
triggerauthentications.keda.k8s.io

is it required for scaling ?

Thanks.

**Comment by tomkerkhove** [comment, marked as answer]
Yes it is, how did you install KEDA?

**Reply by jitendrabehera2** [reply]
I have deployed the KEDA by using YAML declarations.
Download the "keda-1.5.0.zip" file from https://github.com/kedacore/keda/releases, and used the below command to install.

kubectl apply -f deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
kubectl apply -f deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
kubectl apply -f deploy/

**Reply by zroubalik** [reply]
What is the output, if you run these commands again?
```
kubectl apply -f deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
kubectl apply -f deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
```

**Reply by jitendrabehera2** [reply]
customresourcedefinition.apiextensions.k8s.io/scaledobjects.keda.k8s.io unchanged
customresourcedefinition.apiextensions.k8s.io/triggerauthentications.keda.k8s.io unchanged

**Reply by zroubalik** [reply]
So the CRDs are installed, why do you think they are not? (your original question)

**Reply by jitendrabehera2** [reply]
scaledobjects.keda.k8s.io
triggerauthentications.keda.k8s.io

How could we confirm these 2 are installed bcoz we are not able to see these 2 resource

**Reply by tomkerkhove** [reply]
You can use this command:
```shell
❯ kubectl get crd
NAME                                    CREATED AT
healthstates.azmon.container.insights   2020-08-06T07:32:20Z
scaledobjects.keda.k8s.io               2020-08-27T12:00:05Z
triggerauthentications.keda.k8s.io      2020-08-27T12:00:05Z
```

**Reply by jitendrabehera2** [reply]
Thanks a lot for you supporting.

**Reply by tomkerkhove** [reply]
Happy to help - Feel free to mark this as answered.
